DEVELOPMENT_GUIDE.md

```diff
@@ -21,6 +21,41 @@ This will start a new Gitpod workspace, and immediately kick off a build of the
 
 Gitpod is free for 50 hours per month - make sure to stop your workspace when you're done (you can always resume it later, and it won't need to run the build again).
 
+5-Minute Tutorial
+-----------------
+
+This tiny tutorial will show you how to make a small change to SAM CLI.
+
+See the rest of this document for a more thorough introduction.
+
+1. Clone SAM CLI:
+
+    ```bash
+    git clone git@github.com:aws/aws-sam-cli.git
+    cd aws-sam-cli
+    ```
+
+2. Create a Python [virtual environment](https://docs.python.org/3/library/venv.html):
+
+    ```bash
+    python3 -m venv .venv
```

Comment:
I feel it should be consistent using `venv` or `pyenv`. If we want to use `venv` here, we might also want to change all `pyenv` to `venv` in this doc.

Reply:
`pyenv` requires additional dependencies and setup for more complex use cases (e.g. quickly switching between Python versions), which are unnecessary to quickly get started on any system.